.github/actions/find-intermediates-base/action.yml

```diff
@@ -0,0 +1,44 @@
+name: Find Intermediates Base
+description: Finds the most recent commit (from the last 5) with cached build artifacts for a specified Nix flake `packages` output. Requires Nix installed on the runner
+inputs:
+  cache_url:
+    description: The URL of the Nix binary cache to check for previous build artifacts.
+    required: true
+  package_name:
+    description: The `packages.<name>` flake output to incrementally build. Defaults to `default`.
+    required: false
+    default: "default"
```

Comment:
Let's remove this default. Less magic / assumptions to remember.

Reply:
https://github.com/juspay/haskell-incremental-build-template/pull/5/commits/b5d95bbd51acb4b8c320fd1414b1d06c941e932a